lib/blockscore/base.rb

```diff
@@ -1,10 +1,12 @@
 require 'blockscore/connection'
+require 'uri'
 
 module BlockScore
   class Base
     extend Connection
 
     ABSTRACT_WARNING = 'Base is an abstract class, not an API resource'.freeze
+    API_URL = URI.parse('https://api.blockscore.com/').freeze
```

Comment:
In a soon-to-be-committed change you will see I have isolated this URI code to one module. So it should be very easy to swap out if needed.